diesel/src/connection/mod.rs

```diff
@@ -7,6 +8,8 @@ use result::*;
 use types::HasSqlType;
 
 pub use self::transaction_manager::{TransactionManager, AnsiTransactionManager};
+#[doc(hidden)]
+pub use self::statement_cache::StatementCacheKey;
```

Comment:
Is this one of the things where we'd like to have <https://github.com/rust-lang/rust/issues/32409>?